documentation/resources/api-discovery.md

```diff
@@ -4,140 +4,192 @@ order: 2200
-3. Click the `PUT /namespaces/{ns}/datasets` accordion item
-4. Click `Try it out`, enter your namespace, personalize any fields in the Request body, remove the lines pertaining to organization, and click `Execute`
-5. Scroll down and ensure a 200 Response was received
+Cards show basic metadata about the API: the API name (`title`), associated ministry (`organization`), description (`notes`), and available environments. This metadata is defined in a `dataset`.
 
+Despite the name `dataset`, don't think that all API's need to serve *data*. Many API's provide services, for example, the [BC Route Planner API](https://api.gov.bc.ca/devportal/api-directory/740?preview=false) returns vehicle routes between start and end points.
+
+!!!info
+Think of `datasets` as API metadata records.
+!!!
+
+Clicking on the API name opens the full API listing details.
+![BC Route Planner API listing details](assets/api-directory-listing.png)
+
+Here, additional metadata details defined in the `dataset` are visible, as well as the associated product(s) which consumers can request access to.
+
+!!!info
+`products` bundle one or more gateway services which are protected in the same way. They are used to manage consumer access.
+!!!
+
+## Sharing Your API
+
+API Listings in the Directory can be created and managed via:
+
+- **Command Line Interface (CLI)**: See the [Quick Start guide](../guides/owner-journey.md) for instructions on downloading the CLI and logging in.
+- **Directory API**: Go to [Gateway Administration](../resources/gateway-admin.md#directory-api) for links to the Directory API specs and Swagger UI.
+- **Web user interface (UI)**: Visit the [API Services Portal](https://api.gov.bc.ca/) and login to the API Provider zone.
+
+!!! API Services Portal web UI limitations
+At this time, not all steps required to create an API listing are possible through the web UI. We recommend using the CLI.
+!!!
+
+### Environments
+
+All API Directory functionality is available in the `test` environment. Feel free to create an API listing there before adding to the production Directory.
+
+| Environment     | API Services Portal Directory Link                               |
+| --------------- | ---------------------------------------------------------------- |
+| TEST / TRAINING | https://api-gov-bc-ca.test.api.gov.bc.ca/devportal/api-directory |
+| PRODUCTION      | https://api.gov.bc.ca/devportal/api-directory                    |
+
+### Setup a Draft Dataset
+
+First, we need a draft dataset with metadata about your API. This information helps consumers find your API in the Directory. If your API is already listed in the [BC Data Catalogue](https://catalogue.data.gov.bc.ca/), skip ahead to [Link Your Dataset to a Product](#link-your-dataset-to-a-product).
+
+Start by writing up a draft dataset in a local YAML file (if using the CLI) or JSON file (if using the API). Here is the schema, omitting some optional fields:
+
++++ YAML Template (w/ field descriptions)
```

Comment:
One way to find out! Bring into dev.mvp and see what we get?
 
It would be nice to avoid much custom html, though might be an option. If the tabs don't work and there's no easy solution we can split and just list one after the other.  :/